### PR TITLE
add xACT 2.54 (new cask)

### PR DESCRIPTION
**Pull request by zibebe**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by bevanjkay** [comment]
@truehumandesign Can you please format the `zap` as a string instead of an array since there's only one item? (There will be a style check for this soon).

**Comment by truehumandesign** [comment]
> @truehumandesign Can you please format the `zap` as a string instead of an array since there's only one item? (There will be a style check for this soon). 

Done

File: Casks/x/xact.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+cask "xact" do
+  arch arm: "arm64", intel: "x64"
+
```
Comment (bevanjkay) [inline Casks/x/xact.rb:3]: @truehumandesign The `arch`should be removed as it is not used in the Cask file.

**Comment by bevanjkay** [comment]
@truehumandesign Two more quick changes here, and I think we should be good to go.

**Comment by truehumandesign** [comment]
> @truehumandesign Two more quick changes here, and I think we should be good to go.

Done 👍

**Comment by bevanjkay** [comment]
Sorry, that's what happens when I make suggestions from my phone 🤦🏼‍♂️

`:homepage` is what I was looking for. 

Thanks for the quick responses, and sorry for so much back and forth on this one

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**